pkgs/development/libraries/libgsf/default.nix

```diff
@@ -21,12 +23,26 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
 
-  src = fetchurl {
-    url = "mirror://gnome/sources/${pname}/${lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
-    sha256 = "bmwg0HeDOQadWDwNY3WdKX6BfqENDYl+u+ll8W4ujlI=";
+  src = fetchFromGitLab {
```

Comment:
Why switch the source?

Reply:
Oh, I don't remember already what was the reason, maybe tarball was not available, something wrong with it?